projects/react-game.md

```diff
@@ -8,7 +8,9 @@ You've now learned enough React to create a game! This project is to practice fu
 
 You are welcome to rewrite the game you made last week in React or to make a different one.
 
-Here are some [examples](https://bg.reactjs.org/community/examples.html). **Please do not select hangman because of its racist origins.**
+Here are some [examples](https://bg.reactjs.org/community/examples.html). 
+**Please do not select hangman because of its racist origins.**
+**Please avoid choosing the BMI calculator from the list of examples given above, because it is racist and has negative implications related to sizeism**
```

Comment:
Please rephrase: 

> Please avoid choosing the BMI calculator from the list of examples given above, because it has negative implications related to racism and sizeism.